commons/com.b2international.index.lucene/src/com/b2international/index/json/BulkUpdateOperation.java

```diff
@@ -48,10 +57,30 @@ public BulkUpdateOperation(BulkUpdate<T> update, ObjectMapper mapper, Mappings m
 	@Override
 	public void execute(IndexWriter writer, Searcher searcher) throws IOException {
 		final DocumentMapping mapping = mappings.getMapping(update.getType());
+		final String scriptName = update.getScript();
+		final String script = mapping.getScript(scriptName).script();
+		
+		final GroovyShell shell = new GroovyShell();
```

Comment:
We should limit access from scripts to bound variables, simple arithmetic operations, a limited set of utility methods (eg. `java.lang.Math`) when the feature comes out of experimental status.